fosdem-bare-metal-in-the-cloud/source/index.rst

```diff
@@ -0,0 +1,65 @@
+
+
+Bare Metal In The Cloud: Isn’t it Ironic?
+=========================================
+
+*by Dmitry Tantsur and Ilya Etingof, Red Hat*
+
+In this talk
```

Comment:
@dtantsur Let's see what's worth adding/removing prior to getting into the details?